src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHandlerManager.cs

```diff
@@ -174,6 +181,10 @@ private void ProcessItems(IComparable version, IProjectChangeDescription project
             {
                 handler.Handle(version, addedItems, removedItems, isActiveContext, logger);
             }
+
+            var notification = CommandLineNotifications.FirstOrDefault();
+            if (notification != null)
```

Comment:
You could presumably use:

`CommandLineNotifications.FirstOrDefault()?.Value.Invoke(_project.FullPath, addedItems, removedItems);`

here